packages/target-tauri/src-tauri/src/html_window/mod.rs

```diff
@@ -38,10 +38,10 @@ mod punycode;
 
 #[allow(clippy::too_many_arguments)]
 #[tauri::command]
-pub(crate) fn open_html_window(
+pub(crate) async fn open_html_window(
     app: tauri::AppHandle,
-    html_instances_state: tauri::State<HtmlEmailInstancesState>,
-    dc: tauri::State<DeltaChatAppState>,
+    html_instances_state: tauri::State<'_, HtmlEmailInstancesState>,
+    dc: tauri::State<'_, DeltaChatAppState>,
```

Comment:
Here I just did what the compiler suggested. I don't really understand what it means.

Reply:
this is a lifetime

Reply:
I know that those are lifetimes, but I don't get lifetimes XD